scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SecurityContext.java

```diff
@@ -35,7 +35,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/", "/resources/**",
                        "/register").permitAll()
                 // The rest of the our application is protected.
-                .antMatchers("/users/**", "/admin/**").hasRole("ADMIN")
+                .antMatchers("/users/**", "/admin/**","/auth/**").hasRole("ADMIN")
```

Comment:
what else is at `/auth/`? If it's just projects, I would open that up to any user and if there are parts that are only accessible for admins, list those separate.